Blockzilla/BrowserViewController.swift

```diff
@@ -588,6 +616,16 @@ extension BrowserViewController: HomeViewDelegate {
     func homeViewDidPressSettings(homeView: HomeView) {
         showSettings()
     }
+    
+    func shareTrackerStatsButtonTapped() {
+        // TODO: Telemetry
+        
+        let numberOfTrackersBlocked = getNumberOfLifetimeTrackersBlocked()
+        let appStoreUrl = String(format: "https://itunes.apple.com/app/id%@", AppInfo.isKlar ? "1073435754" : "1055677337")
```

Comment:
There has got to be a better way to do this.

Reply:
We will get you adjust links to track this install channel. Stay tuned.